src/esp/metadata/attributes/ObjectAttributes.cpp

```diff
@@ -10,44 +10,41 @@ namespace attributes {
 ObjectAttributes::ObjectAttributes(const std::string& handle)
     : AbstractObjectAttributes("ObjectAttributes", handle) {
   // fill necessary attribute defaults
-  setMass(1.0);
-  setCOM({0, 0, 0});
-  setInertia({0, 0, 0});
-  setLinearDamping(0.2);
-  setAngularDamping(0.2);
+  init("mass", 1.0);
+  init("COM", Mn::Vector3{0.0, 0.0, 0.0});
+  init("inertia", Mn::Vector3{0.0, 0.0, 0.0});
+  init("linear_damping", 0.2);
+  init("angular_damping", 0.2);
 
   setComputeCOMFromShape(true);
 
-  setBoundingBoxCollisions(false);
-  setJoinCollisionMeshes(false);
-  // default to use material-derived shader unless otherwise specified in config
-  // or instance config
-  setShaderType(getShaderTypeName(ObjectInstanceShaderType::Material));
-  // TODO remove this once ShaderType support is complete
-  setForceFlatShading(false);
-  setIsVisible(true);
-  setSemanticId(0);
+  init("use_bounding_box_for_collision", false);
+  init("join_collision_meshes", false);
+
+  init("semantic_id", 0);
```

Comment:
Is this "unknown" or could it be a random class?
Should probably reserve the default for unknown.
Doesn't have to be this PR, just thought I'd mention it.

Reply:
This semantic ID being set to 0 corresponds to the stage.